src/write.c

```diff
@@ -1380,6 +1380,15 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
             avifRWStreamWriteU16(&s, 0x0018);                          // template unsigned int(16) depth = 0x0018;
             avifRWStreamWriteU16(&s, (uint16_t)0xffff);                // int(16) pre_defined = -1;
             writeConfigBox(&s, &item->av1C);
+
+            avifBoxMarker ccst = avifRWStreamWriteFullBox(&s, "ccst", AVIF_BOX_SIZE_TBD, 0, 0);
+            const uint8_t ccstValue = (0 << 7) | // unsigned int(1) all_ref_pics_intra;
+                                      (1 << 6) | // unsigned int(1) intra_pred_used;
+                                      (15 << 2); // unsigned int(4) max_ref_per_pic;
```

Comment:
I think it's fine to use the special value 15 here, but I am just wondering if this value could be set to 7 or 8 for AV1.

Reply:
Yeah it is not a big deal. We can use either. This box is just here for conformance purposes and it's very unlikely that any decoder will actually use it in a meaningful way.